common/pkcs11.h

```diff
@@ -748,27 +835,41 @@ typedef unsigned long ck_mechanism_type_t;
 #define CKM_DSA_PARAMETER_GEN		(0x2000UL)
 #define CKM_DH_PKCS_PARAMETER_GEN	(0x2001UL)
 #define CKM_X9_42_DH_PARAMETER_GEN	(0x2002UL)
+#define CKM_DSA_PROBABLISTIC_PARAMETER_GEN	(0x2003UL)
+#define CKM_DSA_SHAWE_TAYLOR_PARAMETER_GEN	(0x2004UL)
+#define CKM_AES_OFB			(0x2104UL)
+#define CKM_AES_CFB64			(0x2105UL)
+#define CKM_AES_CFB8			(0x2106UL)
+#define CKM_AES_CFB128			(0x2107UL)
+#define CKM_AES_CFB1			(0x2108UL)
+
 #define CKM_VENDOR_DEFINED		((unsigned long) (1UL << 31))
 
 /* Ammendments */
-#define CKM_SHA224 (0x255UL)
-#define CKM_SHA224_HMAC (0x256UL)
-#define CKM_SHA224_HMAC_GENERAL (0x257UL)
-#define CKM_SHA224_RSA_PKCS (0x46UL)
-#define CKM_SHA224_RSA_PKCS_PSS (0x47UL)
-#define CKM_SHA224_KEY_DERIVATION (0x396UL)
-
-#define CKM_CAMELLIA_KEY_GEN (0x550UL)
-#define CKM_CAMELLIA_ECB (0x551UL)
-#define CKM_CAMELLIA_CBC (0x552UL)
-#define CKM_CAMELLIA_MAC (0x553UL)
-#define CKM_CAMELLIA_MAC_GENERAL (0x554UL)
-#define CKM_CAMELLIA_CBC_PAD (0x555UL)
-#define CKM_CAMELLIA_ECB_ENCRYPT_DATA (0x556UL)
-#define CKM_CAMELLIA_CBC_ENCRYPT_DATA (0x557UL)
-
-#define CKM_AES_KEY_WRAP (0x2109UL)
-#define CKM_AES_KEY_WRAP_PAD (0x210aUL)
+#define CKM_SHA224			(0x255UL)
+#define CKM_SHA224_HMAC			(0x256UL)
+#define CKM_SHA224_HMAC_GENERAL		(0x257UL)
+#define CKM_SHA224_RSA_PKCS		(0x46UL)
+#define CKM_SHA224_RSA_PKCS_PSS		(0x47UL)
+#define CKM_SHA224_KEY_DERIVATION	(0x396UL)
+
+#define CKM_CAMELLIA_KEY_GEN		(0x550UL)
+#define CKM_CAMELLIA_ECB		(0x551UL)
+#define CKM_CAMELLIA_CBC		(0x552UL)
+#define CKM_CAMELLIA_MAC		(0x553UL)
+#define CKM_CAMELLIA_MAC_GENERAL	(0x554UL)
+#define CKM_CAMELLIA_CBC_PAD		(0x555UL)
+#define CKM_CAMELLIA_ECB_ENCRYPT_DATA	(0x556UL)
+#define CKM_CAMELLIA_CBC_ENCRYPT_DATA	(0x557UL)
+
+#define CKM_AES_KEY_WRAP		(0x2109UL)
+#define CKM_AES_KEY_WRAP_PAD		(0x210aUL)
+
+#define CKM_RSA_PKCS_TPM_1_1		(0x4001UL)
+#define CKM_RSA_PKCS_OAEP_TPM_1_1	(0x4002UL)
+
+#define CKM_EC_EDWARDS_KEY_PAIR_GEN	(CKM_VENDOR_DEFINED|0x9040UL)
+#define CKM_EDDSA			(CKM_VENDOR_DEFINED|0x9041UL)
```

Comment:
That's also a value to be defined in PKCS#11 3.0

Reply:
Yes, SoftHSM should be updated with values from 3.0